config/src/types.rs

```diff
@@ -75,6 +78,7 @@ impl Default for WalletConfig {
 			tls_certificate_key: None,
 			dark_background_color_scheme: Some(true),
 			keybase_notify_ttl: Some(1440),
+			accept_fee_base: None,
```

Comment:
Shouldn't it be?
```suggestion
			accept_fee_base: default_accept_fee_base(),
```

Reply:
if accept_fee_base is not configured then it falls back to the global default. so these would seem to be equivalent.